Robot_number: 2

state_0: [-1, 0, 4.0,     1.0,0.0,0.0,0.0,     0.0,0.0,0.0]
robot_arm_len: 0.2
robot_m: 1.0


obtscles_pos: [[-2,-3,0]] #,[2,3.2,0.5],[1.0,1.7,0.2]]
obtscles_r: [0.2] #, 0.2, 0.2]

